racket/collects/compiler/cm.rkt

```diff
@@ -2,7 +2,7 @@
 (require "private/cm-minimal.rkt"
          (submod "private/cm-minimal.rkt" cm-internal)
          racket/contract/base
-         racket/place
+         '#%place
```

Comment:
This is ok, but I can't help thinking it's better to avoid `#%` modules in all but the most primitive modules. How about adding `racket/place-channel` or `racket/private/place-channel` to just re-exports place channels from `'#%place`?